src/App.js

```diff
@@ -1,19 +1,22 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { Route, Switch, Redirect } from 'react-router-dom';
+
+import Home from "./components/containers/home/";
+import Keeps from "./components/containers/keeps/";
+
+import Navegation from "./components/component/navegation";
 
 class App extends Component {
   render() {
     return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
-        </header>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
-      </div>
+      <React.Fragment>
+      <Navegation/>
+      <Switch>
+        <Route path="/keeps"  component={Keeps} />
+        <Route path="/" component={Home} />
+        <Redirect to="/" />
```

Comment:
It is a good practice to send the object on the redirect so you can let the user know what happened before the redirect.